homeassistant/util/yaml.py

```diff
@@ -10,6 +10,19 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class ConfigDict(OrderedDict):
```

Comment:
We could move all this init logic into `_ordered_dict` and we don't even have to subclass it.